tests/AbstractProcessTest.php

```diff
@@ -333,4 +406,16 @@ private function getPhpBinary()
 
         return $runtime->getBinary();
     }
+    
+    private function getPhpCommandLine($phpCode)
+    {
+    	$cmd = $this->getPhpBinary() . ' -r ' . escapeshellarg($phpCode);
+    	
+    	if (defined('PHP_WINDOWS_VERSION_BUILD')) {
+    		// Windows madness! for some obscure reason, the whole command lines needs to be
+    		// wrapped in quotes (?!?)
+    		$cmd = '"'.$cmd.'"';
```

Comment:
Another tip from @auroraeosrose: rather than do this, we should use the `bypass_shell` option for `proc_open()`, which is only relevant to Windows and ignored for *nix systems.